packages/component-library/src/MapTooltip/MapTooltip.js

```diff
@@ -73,12 +74,26 @@ const MapTooltip = props => {
           )}
         </div>
       )}
-      {tooltipDataArray.map(el => (
-        <div key={el.name}>
-          <strong>{`${el.name}: `}</strong>
-          {getProperty(tooltipInfo.properties[el.field], el.formatField)}
-        </div>
-      ))}
+      {!isScreenGrid &&
+        tooltipDataArray.map(el => (
+          <div key={el.name}>
+            <strong>{`${el.name}: `}</strong>
+            {getProperty(tooltipInfo.properties[el.field], el.formatField)}
+          </div>
+        ))}
+      {isScreenGrid &&
+        tooltipDataArray.map(el => (
+          <div key={el.name}>
+            <strong>{`${el.name}: `}</strong>
+            {!el.totalField &&
+              getProperty(tooltipInfo[el.field], el.formatField)}
+            {el.totalField &&
+              getProperty(
+                tooltipInfo[el.field] / tooltipInfo[el.totalField],
+                el.formatField
+              )}
```

Comment:
In the section above, it looks like you're calculating the average using `cellWeight` divided by `cellCount`. But couldn't you just set the new `aggregation` prop to `"MEAN"` and achieve the same results? Why hard code this here?